_source/user-guide/kibana/index.md

```diff
@@ -8,24 +8,25 @@ tags:
   - kibana
 contributors:
   - imnotashrimp
+  - shalper
 ---
 
 When you first sign in to Logz.io, you'll find yourself at the Discover page. You can use the Discover page to filter recent logs, search for specific events, and create alerts based on your search queries.
 
-![Kibana UI in Logz.io](https://dytvr9ot2sszz.cloudfront.net/logz-docs/kibana/kibana--discover-annotated.png)
+![Kibana UI in Logz.io](https://dytvr9ot2sszz.cloudfront.net/logz-docs/kibana/kibana--discover-annotated1.png)
 
 Query bar
-: This is where you type your Elasticsearch query
+: This is where you type your search query
 
-Field and account selection
-: Choose the accounts to search and the fields that will get a table column
+Account and field selection
+: Choose the accounts to search and add fields to the preview table
```

Comment:
Not sure why are the fields mentioned here. This is mainly focused on access to the searchable data. Once data appears on the results then naturally fields can be added. It is better we indicated what can be done to have accounts available here and how are the accounts structured and what are the limitations maybe.